helper.fish

```diff
@@ -487,19 +538,26 @@ function buildTarGzPackageHelper
     set name arangodb3
   end
 
-  pushd $WORKDIR/work/ArangoDB/build/install
-  and rm -rf bin sbin
+  pushd $WORKDIR/work
+  and rm -rf targz
+  and mkdir targz
+  and cd $WORKDIR/work/ArangoDB/build/install
+  and cp -a * $WORKDIR/work/targz
+  and cd $WORKDIR/work/targz
+  and rm -rf bin
   and cp -a $WORKDIR/binForTarGz bin
-  and rm -f "bin/*~" "bin/*.bak"
+  and find bin "(" -name "*.bak" -o -name "*~" ")" -delete
   and mv bin/README .
-  and mkdir sbin
-  and mv bin/arangod sbin
-  and strip usr/sbin/arangod usr/bin/{arangobench,arangodump,arangoexport,arangoimp,arangorestore,arangosh,arangovpack}
+  and if test $PACKAGE_STRIP = All
+    strip usr/sbin/arangod usr/bin/{arangobench,arangodump,arangoexport,arangoimp,arangorestore,arangosh,arangovpack}
+  else if test $PACKAGE_STRIP = ExceptArangod
+    strip usr/bin/{arangobench,arangodump,arangoexport,arangoimp,arangorestore,arangosh,arangovpack}
```

Comment:
AFAIK `arangoimp` is only a symlink for `arangoimport` nowadays.
I guess running `strip` on a symlink will also work, but I would find it more intuitive to run `strip` on the original executable. This is a nice-to-have change though.

Reply:
Confirmed: running strip on a symlink also works. So we can ignore this nice-to-have improvement.

Reply:
Done to just avoid misconception in future.